### PR TITLE
[0.12.0] Migration: `next-translate` in Classes

### PR DESCRIPTION
**Pull request by SiravitPhokeed**
## Fixes
- “อาหารที่แพ้” is misspelled in Student List Printout
  Fixes FRO-196 [(link)](https://linear.app/skiso/issue/FRO-196/อาหารที่แฟ้)
- Subtitles and icons are now removed from Lookup Class Card after school
- Lookup Class Card appears to fetch Schedule data even when school is not in session

## Code Changes
- Migrated `classes.json` to `next-translate`
  Resolves FRO-164 [(link)](https://linear.app/skiso/issue/FRO-164/migrate-classrooms-page)
- Scheme Icon prop value was redundant
- All uses of `forwardRef` were missing explicit types

### CODE REVIEW

File: components/classes/LookupClassCard.tsx [hunk 1]
```diff
@@ -51,7 +59,21 @@ const LookupClassCard: StylableFC<{
...
+      schoolSessionState === SchoolSessionState.schedule &&
+      [4, 5].includes(periodNumber)
+    )
+      return LookupClassCardView.lunch;
+    if (period) return LookupClassCardView.upcoming;
+    return LookupClassCardView.finished;
+  })();
+
   useEffect(() => {
+    if (schoolSessionState !== SchoolSessionState.schedule) return;
```
Comment (Sun-52) [inline components/classes/LookupClassCard.tsx:76]: This is good for reducing the number of queries made.

**Review by Sun-52 — APPROVED** [review]
LGTM!

File: components/Layout.tsx [hunk 2]
```diff
@@ -261,9 +261,7 @@ const Layout: FC<
               <NavBarItem
                 icon={
                   <SchemeIcon
-                    colorScheme={
-                      (colorScheme === "dark" ? "light" : "dark") || "light"
-                    }
+                    colorScheme={colorScheme === "dark" ? "light" : "dark"}
```
Comment (pixelpxed) [inline components/Layout.tsx:264]: Great that we reduced some calculations here and there...

**Review by pixelpxed — APPROVED** [review]
LGTM!

**Pull request merged**